include/uv.h

```diff
@@ -1681,10 +1682,15 @@ struct uv_loop_s {
   } active_reqs;
   /* Internal flag to signal loop stop. */
   unsigned int stop_flag;
-  void* reserved[4];
+  uv_loop_cb watcher_queue_updated_cb;
+  void* reserved[3];
```

Comment:
Since this targets master now, please leave 4 reserved fields. We can break the ABI here.